neptune_tensorboard/sync/tensorflow_data_sync.py

```diff
@@ -68,8 +83,13 @@ def _load_single_run(self, path):
                                                  run_monitoring_thread=False,
                                                  handle_uncaught_exceptions=True,
                                                  hostname=hostname or None) as exp:
+
+                accummulator = self._init_accummulator(path, exp)
```

Comment:
We create an accumulator and check if the file describes an experiment - but it's too late, since we had to create an experiment first to check the limits.